File: PHP/Program/Fitur.php
<?php

class Fitur
{
  public function __construct()
  {
  }

  public function view($data)
  {
    echo '
    <center>
      <table border="1" width="80%" cellspacing="0">
        <tr>
          <th>Foto</th>
          <th>Nama</th>
          <th>Nim</th>
          <th>Prodi</th>
          <th>Fakultas</th>
        </tr>
        <tr align="center"> 
          <td><img src="' . $data->getFoto() . '" width="50"></img></td>
          <td>' . $data->getNama() . '</td>
          <td>' . $data->getNim() . '</td>
          <td>' . $data->getProdi() . '</td>
          <td>' . $data->getFakultas() . '</td>
        </tr>
      </table>
    </center>';
  }

  public function __destruct()
  {
  }
}

File: CPP/Program/Fitur.cpp
#include "Mahasiswa.cpp"
#include <iostream>
#include <string>

using namespace std;

class Fitur
{
public:
  Fitur(){};

  void create()
  {
    Mahasiswa mhs;
    string nama, nim, prodi, fakultas;

    cout << "Input Nama : ";
    cin >> nama;
    cout << "Input NIM : ";
    cin >> nim;
    cout << "Input Program Studi : ";
    cin >> prodi;
    cout << "Input Fakultas : ";
    cin >> fakultas;
    cout << "\n";

    mhs.setNama(nama);
    mhs.setNim(nim);
    mhs.setProdi(prodi);
    mhs.setFakultas(fakultas);
  }

  void read()
  {
    Mahasiswa mhs;
    cout << "Nama     : " << mhs.getNama() << endl;
    cout << "Nim      : " << mhs.getNim() << endl;
    cout << "Prodi    : " << mhs.getProdi() << endl;
    cout << "Fakultas : " << mhs.getFakultas() << endl;
  }

  void update()
  {
  }

  void remove()
  {
  }

  ~Fitur(){};
};
